tests/async_model/models.py

```diff
@@ -0,0 +1,6 @@
+from django.db import models
```

Comment:
I don't think we need a new test module (`async_model`) for these few tests, it’s just creating sprawl. (I know you copied from `async_queryset` - I don't think that needs its own test module either, really.)

Better to rename `async_queryset` to `async_models` - or perhaps just merge the queryset and model tests into the `async` folder, with separate test files. That test module is also small.

Reply:
what name should i keep for the separate test files? Will ``test_async_model_methods.py`` and ``test_async_queryset.py`` work ?

Reply:
Yup!